### ISSUE TITLE
Mondo dependency upgrade tracker

### ISSUE THREAD

**Issue by pauljamescleary** [vinyldns/vinyldns#970]
Update all of the dependencies.  Scan what is in our build, and if there is a new version then pull it in.

Note: some dependencies may require code updates to get working.

Happy to track them here in a list and check them off:

- [ ] fs2 update to 2.4.4 or later

**Comment by chetankm-cs** [comment]
Do we only want to update fs2 dependency ?

**Comment by pauljamescleary** [comment]
@chetankm-cs no, but that is a start :). The general idea is to see if there are other dependencies that can be upgraded and do those as well.   Keep in mind: does not have to be the very latest, as some version bumps may break things.

If you take a stab and update others and the build passes (running `sbt validate verify` locally), then you can do 1 or as many as you like.

**Comment by mfirry** [comment]
Will https://github.com/scala-steward-org/scala-steward help?

**Comment by pauljamescleary** [comment]
@mfirry that would be a good idea and worth a shot too :)
